src/ucs/config/parser.c

```diff
@@ -475,6 +506,59 @@ void ucs_config_help_bitmap(char *buf, size_t max, const void *arg)
     __print_table_values(arg, buf + strlen(buf), max - strlen(buf));
 }
 
+int ucs_config_sscanf_flags(const char *buf, void *dest, const void *arg)
+{
+    char *str                     = ucs_strdup(buf, "config_sscanf_flags_str");
+    ucs_config_flags_args_t *args = (ucs_config_flags_args_t*)arg;
+    char *p, *saveptr;
+    int ret, i;
+
+    if (str == NULL) {
+        return 0;
+    }
+
+    ret = 1;
+    *((uint64_t*)dest) = 0;
```

Comment:
I would propose to create a variable to avoid repetitive casts:
`uint64_t *flags = dest;`